cloud/ovirt/ovirt_networks.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2016 Red Hat, Inc.
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+try:
+    import ovirtsdk4 as sdk
+    import ovirtsdk4.types as otypes
+except ImportError:
+    pass
+
+from ansible.module_utils.ovirt import *
+
+
+DOCUMENTATION = '''
+---
+module: ovirt_networks
+short_description: Module to manage logical networks in oVirt
+version_added: "2.3"
+author: "Ondra Machacek (@machacekondra)"
+description:
+    - "Module to manage logical networks in oVirt"
+options:
+    name:
+        description:
+            - "Name of the the network to manage."
+        required: true
+    state:
+        description:
+            - "Should the network be present or absent"
+        choices: ['present', 'absent']
+        default: present
+    datacenter:
+        description:
+            - "Datacenter name where network reside."
+    description:
+        description:
+            - "Description of the network."
+    comment:
+        description:
+            - "Comment of the network."
+    vlan_tag:
+        description:
+            - "Specify VLAN tag."
+    vm_network:
+        description:
+            - "If I(True) network will be marked as network for VM."
+    mtu:
+        description:
+            - "Maximum transmission unit (MTU) of the network."
+extends_documentation_fragment: ovirt
+'''
+
+EXAMPLES = '''
+# Examples don't contain auth parameter for simplicity,
+# look at ovirt_auth module to see how to reuse authentication:
+
+# Create network
+- ovirt_networks:
+    auth: "{{ ovirt_auth }}"
```

Comment:
Please remove auth parameter to be consistent with other modules and examples note